extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -47,6 +47,7 @@ tags: {  # <amp-autocomplete src>
     name: "filter-expr"
     requires_extension: "amp-bind"
   }
+  attrs: { name: "filter-value" }
```

Comment:
Is this line part of this feature?

Reply:
Oh oops, should I make a one line PR for this? I had just realized this was missing while I was thinking about the validator 😅

Reply:
Nah I just wanted to make sure this was needed and not extraneous. It's low-risk of needing to be reverted, so let's save the extra PR and merge it here